### ISSUE TITLE
Mobile App friendly boilerplate

### ISSUE THREAD

**Issue by Tazer** [laardee/serverless-authentication-boilerplate#21]
Hello,

This works great on the web. But it would be cool with a boilerplate for a "mobile app". 
Where you don't use the redirect flows. Cause that's usually handled by the app and then the app will have a token that it would like to send to `serverless-authentication`  

Any plans on that ? :)

**Comment by thejeff77** [comment]
Yeah... same here. Not sure what you're trying to do, but you can use a cognito blueprint which would be a different repo and fit your use case better.

I want to avoid cognito and have custom authorizers instead to save money, but it seems that there are no open source projects for this yet. Want to start a project?

Link to my feature request: https://github.com/laardee/serverless-authentication-boilerplate/issues/66
